.github/workflows/ci.yml

```diff
@@ -0,0 +1,41 @@
+---
+name: OPENWISP WIFI LOGIN PAGES CI BUILD
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches: -master
+
+jobs:
+  build:
+    name: QA-Checks and Tests
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Set up yarn
+        uses: borales/actions-yarn@v2.0.0
+        with:
+          cmd: install
```

Comment:
Is there any way to use cache, similar to what was done in `travis.yml` ?

Reply:
@devkapilbansal Please see again, I have used the cache this time.